docs/server/settings.rst

```diff
@@ -295,25 +295,38 @@ Configuration settings for applications used by Pootle.
   Default::
 
     {
-        'EDIT': 5.0/7,
-        'REVIEW': 2.0/7,
-        'SUGGEST': 0.2,
-        'ANALYZE': 0.1,
+        'SUGGESTION_ADD': 0,
+        'SUGGESTION_ACCEPT': .1,
+        'SUGGESTION_REJECT': .1,
+        'CHECK_MUTED': 0,
+        'CHECK_UNMUTED': 0,
+        'COMMENT_UPDATED': .1,
+        'TARGET_UPDATED': .3,
+        'STATE_TRANSLATED': .6,
+        'STATE_FUZZY': .1,
+        'STATE_UNFUZZY': .1,
+        'STATE_FUZZY_TRANSLATED': .2,
```

Comment:
lets remove all options that are not used - ditto event handlers

we want to make it as simple as possible for users to understand - and as performant as possible